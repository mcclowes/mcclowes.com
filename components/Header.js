const Header = () => {
  return <h1 className="title">mcclowes</h1>
}

export default Header