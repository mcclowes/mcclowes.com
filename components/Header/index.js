import React from "react";
import styled from "styled-components";

const HeaderWrapper = styled.h1`
  font-size: 1.2em;
  font-family: sans-serif;
`;

const Header = () => {
  return <HeaderWrapper className="title">mcclowes</HeaderWrapper>;
};

export default Header;
