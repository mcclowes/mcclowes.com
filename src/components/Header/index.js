import React from "react";
import PropTypes from "prop-types";

import Navigation from "./Navigation";

import Banner from "../Banner";

// ----------------------------------------------------

const Header = props => (
	<div>
		<Navigation { ...props } />

		{props.homepage && <Banner />}

		{props.children}
	</div>
);

Header.propTypes = {
	children: PropTypes.node,
};

// ----------------------------------------------------

export default Header;
