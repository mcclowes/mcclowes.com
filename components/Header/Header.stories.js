import Header from "./index";
import React from "react";

export default { title: "Header" };

export const header = () => {
  return <Header />;
};
