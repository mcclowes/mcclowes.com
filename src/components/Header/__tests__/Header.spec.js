import Header from "../Header";
import React from "react";
import { shallow } from "enzyme";

describe("Header", () => {
  it("renders component", () => {
    const wrapper = shallow(<Header siteTitle="Default Title" />);

    expect(wrapper).toMatchSnapshot();
  });
});
