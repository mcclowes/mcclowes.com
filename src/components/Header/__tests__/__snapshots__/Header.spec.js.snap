// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders component 1`] = `
<div
  style={
    Object {
      "marginBottom": "1.45rem",
    }
  }
/>
`;
